js/reducers/report-reducer.ts

```diff
@@ -272,13 +277,22 @@ function processReportItemRequests(state: ReportState) {
         let interactiveState: any = null;
         let authoredState: any = null;
         let answerValue: any = answer.get("answer");
+        let reportState: any = answer.get("reportState");
```

Comment:
In the context of an open response question, `answer.get("answer")` returns a simple string (e.g., "This answer is yes."). That was causing the `JSON.parse(answerValue)` below to fail. `reportState` is present in `answer` though and contains `authoredState` and `interactiveState`. Does what I've done here make sense? If so, should I try to consolidate and/or refactor all these try/catch blocks? It seems like the errors they're reporting aren't exactly true in the case where a subsequent try might make up for the previous failed try?